guidelines/dxp/how_to_use_css_custom_properties.md

```diff
@@ -1,132 +1,117 @@
-# How to use CCP (CSS Custom Properties)?
+# How to use CSS Custom Properties in Liferay DXP?
 
-## What are CCP?
+## What are CSS Custom Properties?
 
-The CSS Custom Properties (CCP) are CSS variables that can be used and updated directly in the browser without having to be processed by a compiler like SCSS.
+CSS Custom Properties
 
-## When to use CCP?
+-   Are CSS variables that can be used to define styles dynamically
+-   Can be manipulated directly on the client side (Browser)
+-   Can be used to create easily customizable designs
+-   Can be used to create components that adapt to their context
 
-The power of CCP is that it can be used in instances of the component.
+## When to use CSS Custom Properties?
 
-Let's say we have a general `.button` component and its default color is `red`.
+CSS Custom Properties are not supported by Internet Explorer (IE) and early versions of all the popular browsers ([view Compatibility Sheet](https://caniuse.com/css-variables)), so our recommendation is to use them in modern projects that don't need a certain level of browser accessibility.
```

Comment:
The original purpose of putting it at the beginning was to don't waste the reader's time "I don't want to learn something I can't use", but since this is a guideline, I guess we can move it 😄